source/_components/sensor.elta.markdown

```diff
@@ -0,0 +1,49 @@
+---
+layout: page
+title: ELTA Sensor
+description: "Instructions on how to set up Elta sensors within Home Assistant."
+date: 2017-02-05 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: http://www.elta.gr/Portals/0/elta.jpg
+ha_category: Sensor
+ha_release: 0.39
+---
+
+The `elta` platform allows you to track deliveries by the [Hellenic Postal Service (ELTA)](https://www.elta.gr/en-us/home.aspx).
+Track ELTA Hellenic Post Packages on your dashboard get Origin (limited)/destinations(full) tracking information
+
+To enable this sensor, add the following lines to your `configuration.yaml`:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: Elta
+    code: YOUR_TRACKING_CODE
+```
+
+Configuration options for the ELTA Sensor:
+
+- **code** (*Required*): The tracking code of the pacage.
+
+The ELTA platform will only accept tracking code in the below format.
+Two letters followed by nine numbers and ending with two letters (total thirteen characters).
+
+ELTA Hellenic Post number rules :  ( # Letter, * Digit, ! Letter Or Digit )
+  ( R# *** *** *** GR )   ( V# *** *** *** GR )
+  ( A# *** *** *** GR )   ( C# *** *** *** GR )
+  ( E# *** *** *** GR )   ( L# *** *** *** GR )
+  ( N# *** *** *** GR )   ( I# *** *** *** GR )
+
+Troubleshooting:
+The tracking number has to end in “GR” or have destination country Greece.
+
+If the tracking code does not meet the above format, there will be log entry in this format.
+Invalid key YOUR_TRACKING_CODE . Key must be 13 characters.
+
+
+<p class='note warning'>
```

Comment:
Because of EU some Tracking numbers can give generic result (like Border Export or Arrival at destination Country).
I am currently working on a detection script that will ask the origin system (let say Royal Mail <UK>) so it always show detailed tracking information.
It will still need to be in the International tracking format < 2 letters and 9 numbers and 2 letters (e.g. AA123456789AA)>
Therefore the log entry.